util/memory/tracker.go

```diff
@@ -53,7 +53,7 @@ import (
 type Tracker struct {
 	mu struct {
 		sync.Mutex
-		// The children memory trackers. If the Tracker is the Global Tracker, like executor.GlobalDiskUsageTracker,
+		// children memory trackers. If the Tracker is the Global Tracker, like executor.GlobalDiskUsageTracker,
```

Comment:
Why delete `The` here?
And this fix is uncorrelated with `making the inserting errors more easier to understand`,   If it's necessary, I think we could be modified it in another PR.

Reply:
the goword abort the commit unless modified it. is this change too small to create another PR?

Reply:
ok, thank you for replay